pkg/maps/ctmap/ctmap.go

```diff
@@ -245,11 +245,9 @@ func GC(m *bpf.Map, mapName string) int {
 	deleted := 0
 
 	switch mapName {
-	case MapName6:
-	case MapName6Global:
+	case MapName6, MapName6Global:
```

Comment:
Good catch! Otherwise the first one will just do a `break` and do nothing, right?